### PR TITLE
add script to auto-generate changelog from changelog files

### PR DESCRIPTION
**Pull request by goapunk**
**Describe your changes**
Briefly explain what you did and provide context for a clearer understanding.

**Tasks**
- [ ] PR name contains story or task reference
- [ ] Steps to recreate and test the changes
- [x] Documentation (docs and inline)
- [ ] Tests (including n+1 and django_assert_num_queries where applicable)
- [ ] Changelog

### CODE REVIEW

File: changelog/parse_changelogs.py [hunk 1]
```diff
@@ -0,0 +1,37 @@
+#!/bin/python3
+
+import os
+import re
+
+
+def combine_sections_in_folder(folder_path):
```
Comment (goapunk) [inline changelog/parse_changelogs.py:7]: most of this is generated by AI, I only made small changes to make it work correctly

**Review by m4ra — COMMENTED** [review]
Looks good, is it still in WIP? otherwise we can merge it.

**Comment by goapunk** [comment]
> Looks good, is it still in WIP? otherwise we can merge it.

I found some issues, is now ready

**Review by m4ra — APPROVED** [review]
LGTM

**Pull request merged**